orchestra/static/orchestra/todos/constants.es6.js

```diff
@@ -0,0 +1,3 @@
+export const PENDING_STATUS = 0
```

Comment:
i forget---do we not have a way to pass enum values to the frontend in open source Orchestra?

Reply:
I couldnt find an example. Maybe @paopow or @jumasheff know a way?